src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -369,7 +369,7 @@ export class AppComponent implements OnInit, AfterViewChecked {
                     // Precalculate the max height and min height
                     let maxHeight = resultSet.rowCount < self._defaultNumShowingRows
                         ? Math.max((resultSet.rowCount + 1) * self._rowHeight, self.dataIcons.length * 30) + 10
-                        : 'inherit';
+                        : (self.dataIcons.length * 30) + 10;
```

Comment:
it's hard for me to tell from the code if this will have side-effects.  If it fixes a high-pri bug then let's push and do a test pass.  I'm assuming you've already run through pre-checkin validation that this works correctly in the various cases regarding resultset count, row count, and window size.